code/graphics/grstub.cpp

```diff
@@ -48,11 +48,6 @@ int gr_stub_save_screen()
 	return 1;
 }
 
-int gr_stub_tcache_set(int bitmap_id, int bitmap_type, float *u_scale, float *v_scale, int tex_unit = 0)
```

Comment:
...But you said...

> It **wasn't used anywhere** and has been superseded by the material system.
> It would also **interfere with a possible texture array usage**.

???